frontend/src/views/ProfileView.vue

```diff
@@ -31,9 +31,9 @@ const handleShowSetDefaultAccount = (address: string) => {
 const handleHideSetDefaultAccount = (address: string) => {
   showSetDefaultAccount.value[address] = false
 }
-const setCurentUserAddress = (address: string) => {
+const setCurentUserAddress = (address: `0x${string}`) => {
   if (address) {
-    store.currentUserAddress = address
+    store.currentPrivateKey = address
```

Comment:
@bradleySuira, how are you assigning an address to a private key?

Reply:
done, that's because we used address in the past